### ISSUE TITLE
Integration of collaborative documentation

### ISSUE THREAD

**Issue by AbcSxyZ** [decidim/documentation#29]
I would warn about potential unorganized work who will append.
A message on gitter
> take a look at this guide we made, it might be useful to you https://labs.openheritage.eu/pages/participatorytext

You can improve your documentation with this kind of work. Your strategy should have a stronger
focus on community work, how to include them. It look like you're doing it in a small group of persons,
without communication.

Some guys work on documentation for they own structure because there is a lack of doc, it seem you're not trying to reach this resources.

**Comment by microstudi** [comment]
What about making a public call to post resources that community has made? this would make easy the task to collect and maybe create a wiki-style guides

**Comment by AbcSxyZ** [comment]
Yes, create a guide to introduce other developer to the production of the documentation. Having a contribution guideline, explain philosophy of the documentation, explain needs, etc.

**Comment by andreslucena** [comment]
I actually reached to OpenSourcePolitics last week regarding this issue, as they have a [documentation site](https://docs-decidim.opensourcepolitics.eu/), although they don't have a free license ("© Open Source Politics 2020"). I've asked them if it's possible to have it licensed compatible with ours license (gdfl and cc-by-sa) cc @paulinebessoles @virgile-dev 

I've also found https://digidemlab.org/decidim-docs/docs/en/1-discover-decidim.html 

> What about making a public call to post resources that community has made? this would make easy the task to collect and maybe create a wiki-style guides 

That'd be really nice!! 

Regarding the current Admin Manual, there is already a work in progress about translating it from Spanish and updating it at #13 


> You can improve your documentation with this kind of work. Your strategy should have a stronger
focus on community work, how to include them. It look like you're doing it in a small group of persons,
without communication.

There's a saying in Spanish demonstrations that says: "no nos mires, únete". Translated that'd be "do not watch us, join us"

**Comment by AbcSxyZ** [comment]
> There's a saying in Spanish demonstrations that says: "no nos mires, únete". Translated that'd be "do not watch us, join us"

I'm not actually working on decidim, I hope I will do that, it's why I'm watching what is happening.. For sure, your sentence is true. But if you don't show you, they won't watch you, and they won't join you.
